Scripts/Python/ki/__init__.py

```diff
@@ -1639,6 +1640,22 @@ def MessageHistory(self, control, set):
                 control.end()
                 control.refresh()
 
+    #~~~~~~~~~~~~~~~~~#
+    #  KI Text Color  #
+    #~~~~~~~~~~~~~~~~~#
+
+    ## Sets the KI Text Color from the Chronicle.
+    def DetermineTextColor(self):
+
+        vault = ptVault()
+        entry = vault.findChronicleEntry(kChronicleKITextColor)
+        if entry is not None:
+            colorStr = entry.chronicleGetValue()
+            PtDebugPrint(f"xKI.DetermineTextColor(): KI Text Color is: \"{colorStr}\".", level=kWarningLevel)
+            args = colorStr.split(",")
+            if len(args) == 3:
+                self.chatMgr.commandsProcessor.SetTextColor(args[0], args[1], args[2])
```

Comment:
```suggestion
                self.chatMgr.commandsProcessor.SetTextColor(*args)
```

You might also consider logging an error if the format is incorrect.